From: Ben Hutchings <ben@decadent.org.uk>
Date: Fri, 9 Oct 2015 04:01:18 +0100
Subject: [PATCH ethtool 2/2] Fix return type of test_free() prorotype
Bug-Debian: https://bugs.debian.org/747959
Origin: https://git.kernel.org/cgit/network/ethtool/ethtool.git/commit/?id=89a949e67102cf5ea8e906b24850816cae946397

The return type should be void, consistent with the definition and
with the standard free() function.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 internal.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/internal.h b/internal.h
index 156770c..b5ef646 100644
--- a/internal.h
+++ b/internal.h
@@ -148,7 +148,7 @@ void *test_calloc(size_t nmemb, size_t size);
 char *test_strdup(const char *s);
 #undef strdup
 #define strdup(s) test_strdup(s)
-void *test_free(void *ptr);
+void test_free(void *ptr);
 #undef free
 #define free(ptr) test_free(ptr)
 void *test_realloc(void *ptr, size_t size);
